src/sentry/models/apikey.py

```diff
@@ -56,7 +56,7 @@ def handle_async_replication(self, region_name: str, shard_identifier: int) -> N
         )
 
     def __str__(self):
-        return str(self.key)
+        return f"api_key_id={self.id}, status={self.status}"
```

Comment:
Put some fields I thought would be useful to guide the user to find the object in the db